From a82ef709d3c8b5f6db9830c00bcfd9cec5c143ce Mon Sep 17 00:00:00 2001
From: Vaibhav Bedia <vaibhav.bedia@ti.com>
Date: Thu, 2 Feb 2012 14:20:08 +0530
Subject: [PATCH 0757/1092] ARM: OMAP: AM33XX: Keep the CLKDIV32K module
 enabled

One of the 32K clocks for the RTCSS is obtained from
CLKDIV32K module. Till the dependencies are fully sorted
out in the clock tree keep this module enabled.

Signed-off-by: Vaibhav Bedia <vaibhav.bedia@ti.com>
---
 arch/arm/mach-omap2/omap_hwmod_33xx_data.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-omap2/omap_hwmod_33xx_data.c b/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
index 77030e4..ceaf244 100644
--- a/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
+++ b/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
@@ -379,6 +379,7 @@ static struct omap_hwmod am33xx_clkdiv32k_hwmod = {
 			.modulemode	= MODULEMODE_SWCTRL,
 		},
 	},
+	.flags		= (HWMOD_INIT_NO_IDLE | HWMOD_INIT_NO_RESET),
 };
 
 /* 'control' class */
-- 
1.7.11.2

